parsl/executors/workqueue/executor.py

```diff
@@ -43,10 +43,10 @@
 from collections import namedtuple
 
 try:
-    import work_queue as wq
-    from work_queue import WorkQueue
-    from work_queue import WORK_QUEUE_DEFAULT_PORT
-    from work_queue import WORK_QUEUE_ALLOCATION_MODE_MAX_THROUGHPUT
+    import ndcctools.work_queue as wq
```

Comment:
Does this change the lower bound of which version of ndcctools parsl will work with?

Reply:
Yes it should. Where do I update the ndcctools' version? It is not in `requirements.txt`.

Reply:
The lower bound is now ndcctools release 7.5.3